src/ui/component/dropDown/dropDown.tsx

```diff
@@ -0,0 +1,34 @@
+import type { FC } from 'react'
+import { Popover } from '@/ui/component/popover/popover'
+import './dropDown.scss'
+
+export type DropDownProps = {
+  placeholder: string
+  value: string[]
+  field: JSX.Element
+  options: JSX.Element
+}
+
+export const DropDown: FC<DropDownProps> = ({ value, placeholder, field, options }) => {
+  return (
+    <div className="okp4-dataverse-portal-dropdown-main">
+      <Popover
+        align="start"
+        content={options}
+        contentClassName="okp4-dataverse-portal-dropdown-content"
+        sideOffset={8}
+        trigger={
+          <div className="okp4-dataverse-portal-dropdown-field">
+            {value.length > 0 ? (
+              field
+            ) : (
+              <p className="okp4-dataverse-portal-dropdown-field-placeholder">{placeholder}</p>
+            )}
```

Comment:
Can you clarify this? I'm having a bit of trouble understanding the purpose. Thx.

Reply:
The goal is to display a placeholder by default if no values are provided, with a style common to all dropdown variants. But after reading your previous [feedback](https://github.com/okp4/dataverse-portal/pull/444#discussion_r1301138660), using the value props might be a mistake here. We can handle the placeholder logic in the variants, that can still share a common CSS style at this component level.